cea/analysis/uncertainty/Individual_Evaluation.py

```diff
@@ -0,0 +1,164 @@
+"""
+This file helps in evaluating individual generation. This will be useful when you need to change the global variables
+and see how the objective function value changes. 
+
+Do ensure you have the uncertainty.csv which will be obtained by running uncertainty_parameters.py
+
+This is part of the uncertainty analysis
+"""
+__author__ = "Sreepathi Bhargava Krishna"
+__copyright__ = "Copyright 2016, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Sreepathi Bhargava Krishna"]
+__license__ = "MIT"
+__version__ = "0.1"
+__maintainer__ = "Daren Thomas"
+__email__ = "thomas@arch.ethz.ch"
+__status__ = "Production"
+
+def individual_evaluation(generation, level, size):
+
+    """
+    :param generation: Generation of the optimization in which the individual evaluation is to be done
+    :param level: Number of the uncertain scenario. For each scenario, the objectives are calculated
+    :param size: Total uncertain scenarios developed. See 'uncertainty.csv'
+    :return: Function saves the new objectives in a json file 
+    """
+
+    import cea.inputlocator
```

Comment:
follow the structure of the imports of cea.demand_main.demand_calcuation. Imports are at the beginning of the file
also: always do at the beginning of the file:

 from __future import division.

This will avoid errors in numpy dividing by a small number and transforming the output in an int.